runtime/src/app/app.ts

```diff
@@ -263,10 +265,5 @@ export async function hydrate_target(dest: Target): Promise<HydratedTarget> {
 }
 
 export function load_component(component: DOMComponentLoader): Promise<DOMComponentModule> {
-	const promises: Array<Promise<any>> = [component.js()];
-	return Promise.all(promises).then(values => values[0]);
-}
-
-function detach(node: Node) {
-	node.parentNode.removeChild(node);
+	return component.js();
```

Comment:
thanks for cleaning this up. now that it's so much simpler maybe we can just get rid of `load_component`? that'd help reduce the coupling between `app` and `prefetch_routes`

Reply:
Definitely!